tests/topotests/bgp_gr_functionality_topo1/test_bgp_gr_functionality_topo1-4.py

```diff
@@ -1887,9 +1930,11 @@ def test_BGP_GR_TC_53_p1(request):
 
     start_router_daemons(tgen, "r1", ["bgpd"])
 
-    output = tgen.gears["r1"].vtysh_cmd(
+    r1.vtysh_cmd(
         """
         configure terminal
+        debug bgp graceful-restart
```

Comment:
Why are these needed?

Reply:
This test case has been failing in CI (on debian)  but it's passing locally for me (I use ubuntu 22). So added this inorder to debug it. I will update this commit or remove it after I root cause the issue